sensing/radar_static_pointcloud_filter/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+cmake_minimum_required(VERSION 3.5)
+project(radar_static_pointcloud_filter)
+
+# Compile options
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 17)
+  set(CMAKE_CXX_STANDARD_REQUIRED ON)
+  set(CMAKE_CXX_EXTENSIONS OFF)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
```

Comment:
Can you change to [this](https://github.com/autowarefoundation/autoware.universe/blob/main/perception/euclidean_cluster/CMakeLists.txt#L4-L5)? cc @kenji-miyake 

```
find_package(autoware_cmake REQUIRED)
autoware_package()
```

Reply:
fixed at [2aa1387](https://github.com/autowarefoundation/autoware.universe/pull/2362/commits/2aa1387a22e2a14f9bd40152bd5f67cf9b374738)